cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/ApplicationRequestsPlaner.java

```diff
@@ -25,6 +28,7 @@
 public class ApplicationRequestsPlaner extends RequestsPlaner {
 
     private static final Log log = Log.getLog(ApplicationRequestsPlaner.class);
+    private static final Set<String> FIELDS_REQUIRE_RESTART = new HashSet<>(Arrays.asList("meta", "path"));
```

Comment:
We should ensure that the fields are actually present in the ApplicationBean class under these names. (Similar to DiffResult, but we could just disregard the types of the fields)

Reply:
This is not super important and can be postponed to a future apply story.